Extension/src/common.ts

```diff
@@ -1810,3 +1810,21 @@ export function findExePathInArgs(args: CommandString[]): string | undefined {
 export function getVsCodeVersion(): number[] {
     return vscode.version.split('.').map(num => parseInt(num, undefined));
 }
+
+export function equals(array1: string[] | undefined, array2: string[] | undefined): boolean {
```

Comment:
TypeScript confusingly was returning false with array1 == array2 even though the contents of the arrays were identical.